packages/flx/lib/signing.dart

```diff
@@ -165,8 +165,8 @@ ECSignature _asn1ParseSignature(Uint8List signature) {
 }
 
 PublicKey _publicKeyFromPrivateKey(ECPrivateKey privateKey) {
-  ECPoint Q = privateKey.parameters.G * privateKey.d;
-  return new ECPublicKey(Q, privateKey.parameters);
+  ECPoint q = privateKey.parameters.G * privateKey.d;
```

Comment:
i funny think we should make the change. is the name of the variable in the spec we're implementing, as I understand it.